src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -89,11 +90,10 @@
             "          I=input.bam \\<br />" +
             "          O=output.txt" +
             "</pre>"+
-            "Please see <a href='http://broadinstitute.github.io/picard/picard-metric-definitions.html#AlignmentSummaryMetrics'>" +
-            "the AlignmentSummaryMetrics documentation</a> for detailed explanations of each metric. <br /> <br />" +
-            "Additional information about Illumina's quality filters can be found in the following documents on the Illumina website: " +
-            "<ul><li>http://support.illumina.com/content/dam/illumina-marketing/documents/products/technotes/hiseq-x-percent-pf-technical-note-770-2014-043.pdf</li> " +
-            "<li>http://support.illumina.com/content/dam/illumina-support/documents/documentation/system_documentation/hiseqx/hiseq-x-system-guide-15050091-d.pdf</li></ul>" +
+            "For detailed explanations of each output metric, please see the AlignmentSummaryMetrics section of our " +
```

Comment:
I just updated the three programs, CollectHsMetrics, CollectRnaSeqMetrics, and BaitDesigner.  The other tools are todo.